server/src/main/java/org/elasticsearch/cluster/routing/allocation/allocator/DesiredBalanceComputer.java

```diff
@@ -128,7 +128,8 @@ public DesiredBalance compute(
             logger.debug("Recomputing desired balance for [{}]", desiredBalanceInput.index());
         }
 
-        final var routingAllocation = desiredBalanceInput.routingAllocation().mutableCloneForSimulation();
+        // TODO: Maybe make this a setting?
```

Comment:
If we make it configurable to anything other than 1, THROTTLE comes into play and NOT_PREFERRED breaks.

We already throttling rebalance() to 2 relocations, per the ConcurrentRebalanceAllocatonDecider's special isSimulating() behavior. I don't believe it's a great risk to reduce from 2 relocations per call to 1.

Reply:
Another one, the change is not relevant anymore